docs/UserGuide.md

```diff
@@ -211,7 +212,7 @@ of each entity in the list.
 **Function**: Edits a dog/owner/program from Pawbook.<br>
 **Usage**: This command allows you to edit the information of the entity (dog/owner/program) in your Pawbook. Let's say 
 you just added an entry for a dog but you realised that you filled in the owner information wrongly. The edit command 
-allows you to quickly correct that mistake.  
+allows you to quickly correct that mistake. Note that the date-time format used for sessions is dd-mm-yyyy hh:mm.
```

Comment:
This part also needs to be edited, to keep things short and succinct, it would be good to just have a link that redirected users to the add portion for details on the parameters to keep things simple and not repeat yourself.